### DISCUSSION TITLE
UCUM translation

### DISCUSSION THREAD

**Discussion by hielkema** [ucum-org/ucum#265] [General]
Dear all, we have a partial translation to UCUM that we would like to extend. Does Regenstrief have a mechanism for maintaining and publishing a UCUM translations, or do you plan to evolve such a mechanism? 
If not, is there a recommended method or format for other organisations to publish a translation? Has any other country done so? I saw on the UCUM site that there is a Chinese translation and a partial German translation: how are they distributed?

Best wishes, Feikje Hielkema-Raadsveld, Nictiz, Netherlands

**Comment by aflackey** [comment]
Hi,
 We are considering how to operationalize linguistic translations for UCUM, but currently do not have a mechanism for doing so. So, you can use your translation locally, but at this time we won't be able to formalize and distribute it.

**Comment by hielkema** [comment]
Hi April,

Thank you for your reply. Would it then be okay for us to publish those translations ourselves, of course including the LOINC license? We would like to ensure that all LOINC users in the Netherlands use the same LOINC translations. If Regenstrief at present is not able to distribute them, then Nictiz as national terminology center of the Netherlands is the next logical authority to distribute them.
Best wishes,
Feikje Hielkema-Raadsveld, expert medical terminology
Nictiz, Dutch national competence centre for electronic exchange of health and care information
View our last updates<https://www.nictiz.nl/> on terminology<https://www.nictiz.nl/standaardisatie/terminologiecentrum/> and the Dutch Labcodeset<https://www.nictiz.nl/standaardisatie/terminologiecentrum/nederlandse-labcodeset/>.
Postbus 19121 | 2500 CC Den Haag | Oude Middenweg 55 | 2491 AC Den Haag M 06-43775314 | ***@***.*** | www.nictiz.nl | linkedin<http://www.linkedin.com/in/feikje-hielkema-raadsveld-486454b>

Van: aflackey ***@***.***>
Verzonden: donderdag 26 januari 2023 18:37
Aan: ucum-org/ucum ***@***.***>
CC: Feikje Hielkema ***@***.***>; Author ***@***.***>
Onderwerp: Re: [ucum-org/ucum] UCUM translation (Discussion #265)


Hi,
We are considering how to operationalize linguistic translations for UCUM, but currently do not have a mechanism for doing so. So, you can use your translation locally, but at this time we won't be able to formalize and distribute it.

**Reply by aflackey** [reply]
Hello,
Yes, you could publish your UCUM translation locally. Do note that you would need to have the UCUM license attribution, as this is distinct from the LOINC license. The UCUM license can be found at https://ucum.org/license, and the Appendix provides details on applying the attribution statement.

**Comment by hielkema** [comment]
Hi April,

Thank you for confirming! We will include the UCUM license attribution.

Best wishes,
Feikje Hielkema-Raadsveld, expert medical terminology
Nictiz, Dutch national competence centre for electronic exchange of health and care information
View our last updates<https://www.nictiz.nl/> on terminology<https://www.nictiz.nl/standaardisatie/terminologiecentrum/> and the Dutch Labcodeset<https://www.nictiz.nl/standaardisatie/terminologiecentrum/nederlandse-labcodeset/>.
Postbus 19121 | 2500 CC Den Haag | Oude Middenweg 55 | 2491 AC Den Haag M 06-43775314 | ***@***.*** | www.nictiz.nl | linkedin<http://www.linkedin.com/in/feikje-hielkema-raadsveld-486454b>

Van: aflackey ***@***.***>
Verzonden: vrijdag 3 februari 2023 21:18
Aan: ucum-org/ucum ***@***.***>
CC: Feikje Hielkema ***@***.***>; Author ***@***.***>
Onderwerp: Re: [ucum-org/ucum] UCUM translation (Discussion #265)


Hello,
Yes, you could publish your UCUM translation locally. Do note that you would need to have the UCUM license attribution, as this is distinct from the LOINC license. The UCUM license can be found at https://ucum.org/license, and the Appendix provides details on applying the attribution statement.

**Comment by BfArM-LOINC-UCUM** [comment]
Hello Feikje, 

in Germany we have published a partial translation of UCUM as well: https://www.bfarm.de/SharedDocs/Downloads/DE/Kodiersysteme/ucum/ucum-valueset-de-ehdsi-20230728.html
The valueset corresponds to the Common UCUM unit valueset: https://github.com/ucum-org/ucum/tree/main/common-units
We supplemented a short display name which is similar to the UCUM code, but adjusted to conventions in Germany. 
Usage of copyright information and references to LOINC licence and UCUM licence have been clarified with the Regenstrief Institute.

Kind regards,
Jeremy

**Reply by hielkema** [reply]
Hi Jeremy, Thanks, that is good to know! And a good idea to take the common UCUM valueset. We should plan that in for next year :-)

**Comment by SRKid** [comment]
I agree, but would like to add one important comment.  Any translation should be restricted to the UCUM's descriptive text, "name", and "type of quantity" -- but NOT the actual UCUM codes themselves (e.g., "print | c/s | c/i" ).  For example, the UCUM concept for "year" should always have the "print | c/s | c/i" showing  "a | a | ANN" even though the English word "year" might be translated into the German word "jahr" or the Japanese word "toshi" for the UCUM "name".  The UCUM concept for "year" should NEVER have a "print | c/s | c/i"  pseudo-translation of the UCUM codes showing something like "j | j | JAH" (for "jahr") or "t | t | TOS" (for "toshi").
